<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Timer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Select playtime and press START GAME!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>font-size: 14px; font-weight: bold;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarPlayer1">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBarPlayer2">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <widget class="QPushButton" name="time120sButton">
       <property name="text">
        <string>120 sec</string>
       </property>
      </widget>
      <widget class="QPushButton" name="time5minButton">
       <property name="text">
        <string>5 min</string>
       </property>
      </widget>
     </layout>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>START GAME</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>STOP GAME</string>
       </property>
      </widget>
     </layout>
     <widget class="QPushButton" name="switchPlayer1">
      <property name="text">
       <string>Switch Player</string>
      </property>
     </widget>
    </layout>
   </widget>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
